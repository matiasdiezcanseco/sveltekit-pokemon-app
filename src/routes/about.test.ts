import About from './about.svelte'
import { describe, test } from 'vitest'
import { render, screen } from '@testing-library/svelte'

describe('Nav', () => {
	test('should match snapshot ', () => {
		const wrapper = render(About as any)
		expect(wrapper).toMatchSnapshot()
	})
})
