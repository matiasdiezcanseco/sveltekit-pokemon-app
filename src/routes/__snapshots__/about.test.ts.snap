// Vitest Snapshot v1

exports[`Nav > should match snapshot  1`] = `
{
  "component": About$ {
    "$$": {
      "after_update": [],
      "before_update": [],
      "bound": {},
      "callbacks": {},
      "context": Map {},
      "ctx": [],
      "dirty": [
        -1,
      ],
      "fragment": {
        "c": [Function],
        "d": [Function],
        "i": [Function],
        "l": [Function],
        "m": [Function],
        "o": [Function],
        "p": [Function],
      },
      "not_equal": [Function],
      "on_destroy": [
        [Function],
      ],
      "on_disconnect": [],
      "on_mount": [],
      "props": {},
      "root": <div>
         
        <h1
          class="text-4xl text-center my-8 uppercase"
        >
          About
        </h1>
         
        <p>
          Sveltekit demo that uses the Pokemon API to make a pokedex.
        </p>
      </div>,
      "skip_bound": false,
      "update": [Function],
    },
  },
  "container": <body>
    <div>
       
      <h1
        class="text-4xl text-center my-8 uppercase"
      >
        About
      </h1>
       
      <p>
        Sveltekit demo that uses the Pokemon API to make a pokedex.
      </p>
    </div>
  </body>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
